File: Progettazione del Software/Laboatorio/Programmi/Esercitazione 2/src/esercizio_3/Esercizio_3.java
package esercizio_3;

import java.util.*;

public class Esercizio_3 {
	public static void main (String[] args) {
		
		Scanner s = new Scanner(System.in);
		
		int anno = Integer.parseInt(s.nextLine());
		
		if(anno % 4 == 0 && anno % 100 != 0 || anno % 400 == 0) {
			System.out.println(anno + " è bisestile.");
		}
		else {
			System.out.println(anno + " non è bisestile.");
		}
	}
}

File: Fondamenti di Informatica 1/Esercitazioni/Esercitazione 2/Lab02/esercizio3.py
a=int(input("Inserisci un anno a: "))
if ((a%4 == 0 and a%100 != 0) or a%400==0):
    print("L'anno "+str(a)+" è bisestile")
else:
    print("L'anno "+str(a)+" non è bisestile")
